task/09-functions-n-closures-tasks.js

```diff
@@ -150,7 +205,20 @@ function logger(func, logFunc) {
  *   partialUsingArguments(fn, 'a','b','c','d')() => 'abcd'
  */
 function partialUsingArguments(fn) {
-    throw new Error('Not implemented');
+    /*
+    const newArgs = Array.from(arguments).slice(1)
+
+    return function () {
+        return fn.apply(null, newArgs.concat(Array.from(arguments)))
+    }
+    */
+
+    //We also could use Array.apply(null, arguments) instead of Array.from(arguments)
+    const newArgs = Array.from(arguments).slice(1)
+
+    return function () {
+        return fn(...newArgs, ...Array.from(arguments))
+    }
```

Comment:
```suggestion
    return function (...args) {
        return fn(...newArgs, ...args)
    }
```

Also one of the possible solutions